packages/vue-app/template/store.js

```diff
@@ -94,10 +94,10 @@ function resolveStoreModules (moduleData, filename) {
   // If src is a known Vuex property
   if (VUEX_PROPERTIES.includes(moduleName)) {
     const property = moduleName
-    const storeModule = getStoreModule(store, namespaces, { isProperty: true })
+    const moduleStore = getStoreModule(store, namespaces, { isProperty: true })
```

Comment:
What about `propertyStoreModule` ?

Reply:
Why not! It's better actually haha
Gonna change that soon

Reply:
Corrected :)